src/Microsoft.ML.Data/Evaluators/Metrics/ClusteringMetrics.cs

```diff
@@ -7,17 +7,17 @@
 namespace Microsoft.ML.Data
 {
     /// <summary>
-    /// The metrics generated after evaluating the clustering predictions.
+    /// The metrics generated after evaluating the clustering predictions (unsupervised learning algorithm).
     /// </summary>
     public sealed class ClusteringMetrics
     {
         /// <summary>
-        /// Normalized Mutual Information
-        /// NMI is a measure of the mutual dependence of the variables.
-        /// <a href="http://en.wikipedia.org/wiki/Mutual_information#Normalized_variants">Normalized variants</a> work on data that already has cluster labels.
-        /// Its value ranged from 0 to 1, where higher numbers are better.
+        /// Normalized Mutual Information is a measure of the mutual dependence of the variables.
```

Comment:
>Normalized Mutual Information is a measure of the mutual dependence of the variables. [](start = 12, length = 85)

Normalized Mutual Information is a measure of the mutual dependence between variables. In the case of clustering, it measures _____ | it tells us _____. This metric is only calculated if the Label column is provided.